SocialCareCaseViewerApi.Tests/V1/Controllers/ResidentControllerTests/UpdateResidentTests.cs

```diff
@@ -35,6 +35,16 @@ public void UpdatePersonReturns200WhenSuccessful()
             result?.StatusCode.Should().Be(204);
         }
 
+        [Test]
+        public void PatchPersonReturns204WhenSuccessful()
```

Comment:
Would be good to add tests for other status code behaviour as well, like what happens when person with that ID does not exist. Or what happens when request is invalid.

Reply:
Added PatchPersonReturns404WhenPersonNotFound()